src/lib.rs

```diff
@@ -22,6 +22,7 @@ pub mod int;
 pub mod mode;
 pub mod oid;
 pub mod ostring;
+pub mod rescharstring;
```

Comment:
Perhaps name the module `cstring` to be consistent with `ostring` and `bstring`.

Reply:
Done (coming)